test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ModelBindingHelperTest.cs

```diff
@@ -39,16 +36,15 @@ public class ModelBindingHelperTest
                 null,
                 Mock.Of<HttpContext>(),
                 new ModelStateDictionary(),
-                metadataProvider.Object,
+                metadataProvider,
                 GetCompositeBinder(binder.Object),
                 Mock.Of<IValueProvider>(),
-                new DefaultObjectValidator(Mock.Of<IValidationExcludeFiltersProvider>(), metadataProvider.Object),
+                new DefaultObjectValidator(Mock.Of<IValidationExcludeFiltersProvider>(), metadataProvider),
                 Mock.Of<IModelValidatorProvider>());
 
             // Assert
             Assert.False(result);
             Assert.Null(model.MyProperty);
-            metadataProvider.Verify();
```

Comment:
Why was this removed? (I suspect because this seems to be not very relevant to the test).